database/db_helpers.js

```diff
@@ -0,0 +1,26 @@
+const knex = require('knex');
+const knexConfig = require('../knexfile');
+
+const db = knex(knexConfig.development);
+
+module.exports = {
+    getDishes: () => {
+        return db('dishes');
+    },
+
+    getDish: id => {
+        return db('dishes').where({ id: id });
+    },
+
+    getRecipes:() => {
```

Comment:
I believe `getRecipes` should return all recipes along with the dishes they belong to, so that would require some `join` table in the method.